openpathsampling/storage/stores/collectivevariable.py

```diff
@@ -150,18 +150,4 @@ def cache_all(self):
         Fill the caches of all CVs
 
         """
-        map(self.fill_cache, self)
-
-    def fill_cache(self, cv):
-        """
-        Fill the cache of a specific CV
-
-        Parameters
-        ----------
-        cv : :obj:`openpathsampling.CollectiveVariable`
-            the CV from which you want the cache to be filled
-
-        """
-        store = self.storage.snapshots.cv_list.get(cv)
-        if store is not None:
-            store.fill_cache()
+        pass
```

Comment:
Sorry, you actually found a _bug_ or not optimal setup. I fixed this and now it is much faster...